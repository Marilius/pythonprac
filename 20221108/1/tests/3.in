a = DivStr("XcDlaoalaofQWEasdssssERTdfgRTY")
print(*a // 5)
print(a % 5)
print(*a % 43 // 14)
print(a.lower() % 3)
print(*a[1:7] // 3)
print(a % 5 + DivStr() + a % 6)