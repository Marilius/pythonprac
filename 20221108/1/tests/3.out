XcDlao alaofQ WEasds sssERT dfgRTY
XcDlaoalaofQWEasdssssERTdfgRTY
Xc Dl ao al ao fQ WE as ds ss sE RT df gR
xcdlaoalaofqweasdssssertdfgrty
cD la oa
XcDlaoalaofQWEasdssssERTdfgRTYXcDlaoalaofQWEasdssssERTdfgRTY